src/host/server.h

```diff
@@ -107,7 +107,7 @@ class CONSOLE_INFORMATION :
 
     ConsoleImeInfo ConsoleIme;
 
-    Microsoft::Console::VirtualTerminal::MouseInput terminalMouseInput;
+    Microsoft::Console::VirtualTerminal::TerminalInput terminalMouseInput;

```

Comment:
actually no, isn't this quite bad? now there's two different `TerminalInput`s in use in conhost simultaneously... one we're using just for key things, and one we're using just for mouse things ...

Reply:
the second one is on the input buffer.